plugins/node/opentelemetry-instrumentation-fastify/src/instrumentation.ts

```diff
@@ -65,10 +65,10 @@ export class FastifyInstrumentation extends InstrumentationBase {
 
   init() {
     return [
-      new InstrumentationNodeModuleDefinition<any>(
+      new InstrumentationNodeModuleDefinition(
         'fastify',
         ['^3.0.0', '^4.0.0'],
-        (moduleExports, moduleVersion) => {
+        (moduleExports: any, moduleVersion) => {
```

Comment:
I notice this is here but not anywhere else. Is that intentional? Do we want to explicitly type out any for the moduleExports?

Reply:
good catch, not sure how this got here. removed 👍